apps/modernization-ui/src/apps/page-builder/page/management/edit/tabs/ManageTabs/ManageTabs.tsx

```diff
@@ -95,24 +115,53 @@ export const ManageTabs = ({ pageId, tabs, onAddSuccess }: Props) => {
                 }
                 modalBody={
                     <div className={styles.modalBody}>
-                        {message && (
+                        {message ? (
                             <AlertBanner type={message.type} expiration={message.expiration}>
                                 {message.message}
                             </AlertBanner>
-                        )}
-                        {addEdit ? <AddEditTab tabData={null} onChanged={setNewTab} /> : null}
+                        ) : null}
```

Comment:
Question:  `{message && (...)}` was converted to `{message ? (...) : null}`, which is functionally the exact same. Is there a preference for one over the other?

Reply:
I just read it was a better practice to have the ternary, it would be more extendible if other conditions arise and it has better legibility for other devs 🤷